x-pack/test/functional/apps/spaces/solution_view_flag_enabled/solution_tour.ts

```diff
@@ -0,0 +1,133 @@
+  const testSubjects = getService('testSubjects');
+  const spacesService = getService('spaces');
+  const browser = getService('browser');
+  const es = getService('es');
+  const log = getService('log');
+
+  describe('space solution tour', () => {
+    let version: string | undefined;
+
+    const removeGlobalSettings = async () => {
+      version = version ?? (await kibanaServer.version.get());
+      version = version.replace(/-SNAPSHOT$/, '');
+
+      log.debug(`Deleting [config-global:${version}] doc from the .kibana index`);
+
+      await es
+        .delete(
+          { id: `config-global:${version}`, index: '.kibana', refresh: true },
+          { headers: { 'kbn-xsrf': 'spaces' } }
+        )
+        .catch((error) => {
+          if (error.statusCode === 404) return; // ignore 404 errors
+          throw error;
+        });
+    };
+
+    before(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+    });
+
+    after(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+    });
+
+    describe('solution tour', () => {
+      let _defaultSpace: Space | undefined = {
+        id: 'default',
+        name: 'Default',
+        disabledFeatures: [],
+      };
+
+      const updateSolutionDefaultSpace = async (solution: SolutionView) => {
+        log.debug(`Updating default space solution: [${solution}].`);
+
+        await spacesService.update('default', {
+          ..._defaultSpace,
+          solution,
+        });
+      };
+
+      before(async () => {
+        _defaultSpace = await spacesService.get('default');
+
+        await PageObjects.common.navigateToUrl('management', 'kibana/spaces', {
+          shouldUseHashForSubUrl: false,
+        });
+
+        await PageObjects.common.sleep(1000); // wait to save the setting
+      });
+
+      afterEach(async () => {
+        await updateSolutionDefaultSpace('classic'); // revert to not impact future tests
+      });
+
+      it('does not show the solution tour for the classic space', async () => {
+        await testSubjects.missingOrFail('spaceSolutionTour', { timeout: 3000 });
+      });
+
+      it('does show the solution tour if the default space has a solution set', async () => {
+        await updateSolutionDefaultSpace('es'); // set a solution
+        await PageObjects.common.sleep(500);
+        await removeGlobalSettings(); // Make sure we start from a clean state
+        await browser.refresh();
+
+        await testSubjects.existOrFail('spaceSolutionTour', { timeout: 3000 });
+
+        await testSubjects.click('closeTourBtn'); // close the tour
+        await PageObjects.common.sleep(1000); // wait to save the setting
+
+        await browser.refresh();
+        await testSubjects.missingOrFail('spaceSolutionTour', { timeout: 3000 }); // The tour does not appear after refresh
+      });
+
+      it('does now show the solution tour after updating the default space from classic to solution', async () => {
```

Comment:
?
```suggestion
      it('does not show the solution tour after updating the default space from classic to solution', async () => {
```

Reply:
Addressed in https://github.com/elastic/kibana/pull/194926/commits/91a8d7165316a385d42e131272e1266af1cb9da9